packages/ma-components/local/page-header/page-header.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from 'react';
+import './page-header.css';
+import '@utrecht/link-css/dist/index.css';
+import '@utrecht/textbox-css/dist/index.css';
+import '@utrecht/button-css/dist/index.css';
+
+export interface PageHeaderProps {
+  leftGroup?: ReactElement;
+  centerGroup?: ReactElement;
+  rightGroup?: ReactElement;
+  navigationBar?: ReactElement;
```

Comment:
Ik vraag me af of het zo handig is om telkens `ReactElement` in plaats van `ReactNode`  te gebruiken. Lost het iets op? `ReactNode` zou ook `ReactElement[]` toestaan, en dat kan soms wel handig zijn. Kortom: omdat ik zelf nog geen voordeel zie, heb ik het idee dat we ons onnodig beperken en later weer wijzigingen op de hals halen.